src/startupscriptgenerator/src/python/scriptgenerator.go

```diff
@@ -226,15 +224,6 @@ func (gen *PythonStartupScriptGenerator) buildGunicornCommandForModule(module st
 	// Default to App Service's timeout value (in seconds)
 	args := "--timeout 600 --access-logfile '-' --error-logfile '-'"
 
-	pythonEnableGunicornMultiWorkers := common.GetBooleanEnvironmentVariable(consts.PythonEnableGunicornMultiWorkersEnvVarName)
```

Comment:
Why do you want to revert all of this? You can probably just skip the test? Also does this test fail on your machine too?

Reply:
I'm testing it on my machine now but the VM is running extremely slow and don't know the result yet. So gonna to make it skip the test and then fix the test later.